app/src/main/java/com/limelight/AppView.java

```diff
@@ -386,6 +398,20 @@ public void run() {
             case CANCEL_ID:
                 return true;
 
+            case VIEW_DETAILS_ID:
+                AlertDialog.Builder builder = new AlertDialog.Builder(this);
+                builder.setMessage("App Id: " + app.app.getAppId())
+                        .show();
+                return true;
+
+            case CREATE_SHORTCUT_ID:
+                ImageView appImageView = info.targetView.findViewById(R.id.grid_image);
+                Bitmap appBits = ((BitmapDrawable)appImageView.getDrawable()).getBitmap();
+                if(!shortcutHelper.createPinnedGameShortcut(uuidString + Integer.valueOf(app.app.getAppId()).toString(), appBits, computer, app.app)) {
```

Comment:
Please use `if (` instead of `if(`. There are several of these throughout this PR.